packagedb/api.py

```diff
@@ -11,39 +11,72 @@
 
 import django_filters
 from django.core.exceptions import ValidationError
-from django.db.models import OuterRef, Q, Subquery
-from django_filters.filters import Filter, OrderingFilter
+from django.db.models import OuterRef
```

Comment:
Why all these import changes?

Reply:
No changes in the import, just fixed the import order using isort.